starter-code/pizza.js

```diff
@@ -1 +1,137 @@
 // Write your Pizza Builder JavaScript in this file.
+// PEPPERONNI BUTTON CLICK
+$(".btn-pepperonni").click(function() {
```

Comment:
an easier way to write this would be 
``` 
$(".btn-pepperonni").click(function() {
  $(".btn-pepperonni").toggleClass("active");
  $(".pep").toggle();
  $(".price li:contains('pepperonni')").toggle();
  updatePrice();
});
```

.toggle() is a built in function of jQuery and it allows classes us to switch been having the class there and then not having the class